apps/keyboard/js/shared/promise_storage.js

```diff
@@ -40,7 +40,7 @@ PromiseStorage.prototype.start = function() {
   return this._getDatabase().catch(function(e) {
     this._openPromise = null;
 
-    throw e;
+    e && console.error(e);
```

Comment:
You would hide other errors here, as shown by the test case. The goal of this patch is only about stop throwing/rejecting for the non-exist Indexed DB case.